home-accountant-web/src/main/webapp/WEB-INF/mvc-dispatcher-servlet.xml

```diff
@@ -15,13 +15,14 @@
 
     <mvc:resources mapping="/**" location="/" cache-period="31556926"/>
     <mvc:annotation-driven/>
-    <context:component-scan base-package="com.godeltech.ha.controller"/>
+    <context:component-scan base-package="com.godeltech.ha.controller,com.godeltech.ha.service"/>
     <context:annotation-config/>
 
-    <bean id="mavenVersionService" class="com.godeltech.ha.service.MavenVersionService" scope="singleton">
+    <bean id="mavenVersion" class="com.godeltech.ha.model.MavenVersion">
     </bean>
 
-    <bean id="mavenVersion" class="com.godeltech.ha.model.MavenVersion">
+    <bean id="mavenVersionService" class="com.godeltech.ha.service.MavenVersionService" scope="singleton">
```

Comment:
@MSapiton if you use @Service annotation in order to specify the service then there is no need to specify bean in xml. please remove.